packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -314,10 +315,7 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
       }
 
       if (s) {
-        return {
-          code: s.toString(),
-          map: config.build.sourcemap ? s.generateMap({ hires: true }) : null
-        }
+        return transformStableResult(s, importer, config)
```

Comment:
Maybe we should revert this change? This is a build-only plugin so we don't really need to take the line structure into account.